common/commands/src/validator/account/send_multiple.rs

```diff
@@ -0,0 +1,234 @@
+    println!("Transaction result code: {}", &res.tx_result.code.value());
+    println!("Transaction hash: {}", &res.hash);
+
+    if let Some(output_filename) = args.output {
+        println!("\nWriting output log to {}", output_filename);
+
+        if let Err(e) = write_output_file(rows, res, &output_filename) {
+            error!(
+                "Failed to write output file {} with error {}",
+                output_filename, e
+            );
+        }
+    }
+}
+
+fn write_output_file(
+    rows: InputFileReader,
+    res: Response,
+    output_filename: &String,
+) -> Result<(), anyhow::Error> {
+    let mut file = fs::OpenOptions::new()
+        .create(true)
+        .append(true)
+        .open(output_filename)?;
+
+    let now = SystemTime::now();
+    let now: DateTime<Utc> = now.into();
+    let now = now.to_rfc3339();
+
+    let data = rows
+        .rows
+        .iter()
+        .map(|row| {
+            format!(
+                "{},{},{},{},{}",
+                row.address, row.amount.amount, row.amount.denom, now, res.hash
+            )
+        })
+        .collect::<Vec<String>>()
+        .join("\n");
+
+    Ok(file.write_all(format!("{}\n", data).as_bytes())?)
+}
+
+#[derive(Debug)]
+pub struct InputFileRow {
+    pub address: AccountId,
+    pub amount: CosmWasmCoin,
+}
+pub struct InputFileReader {
+    pub rows: Vec<InputFileRow>,
+}
+
+impl InputFileReader {
+    pub fn new(path: &str) -> Result<InputFileReader, anyhow::Error> {
+        let mut rows: Vec<InputFileRow> = vec![];
+        let file_contents = fs::read_to_string(path)?;
+
+        let lines: Vec<String> = file_contents.lines().map(String::from).collect();
+        for line in lines {
+            let tokens: Vec<_> = line.split(&[',']).filter(|k| !k.is_empty()).collect();
+            if tokens.len() < 3 {
+                warn!(
+                    "'{}' does not have enough columns, expecting <address>,<amount>,<denom>",
+                    line
+                );
+                continue;
+            }
+
+            // try parse amount to u128
+            let amount = Uint128::from_str(tokens[1]);
+            if amount.is_err() {
+                warn!("'{}' has an invalid amount", line);
+                continue;
+            }
+            let mut amount = amount.unwrap();
+            let mut denom: String = tokens[2].into();
+
+            // multiply when a whole token amount, e.g. 50nym (50.123456nym is not allowed, that must be input as 50123456unym)
+            if !denom.starts_with('u') {
+                amount.mul_assign(Uint128::new(1_000_000u128));
+                denom = format!("u{}", denom);
+            }
+
+            let amount = CosmWasmCoin::new(amount.into(), denom);
+
+            // try parse address
+            let address = AccountId::from_str(tokens[0]);
+            if let Err(e) = address {
+                warn!("'{}' has an invalid address: {}", line, e);
+                continue;
```

Comment:
imo you should abandon the whole thing if there are any failures so that you're not going to send a part of your transaction without knowing about it